source/beamline/BioXAS/BioXASSSRLMonochromatorMaskState.cpp

```diff
@@ -0,0 +1,100 @@
+#include "BioXASSSRLMonochromatorMaskState.h"
+#include "actions3/AMActionSupport.h"
+#include "actions3/AMListAction3.h"
+
+BioXASSSRLMonochromatorMaskState::BioXASSSRLMonochromatorMaskState(const QString &name, QObject *parent) :
+	AMSingleEnumeratedControl(name, "", parent)
```

Comment:
It could be, for sure. It is an AMSingleEnumeratedControl because the mono mask state began as a read-only control that mapped the blades status pv to either Open or Closed. The choice definitely made sense in this case. This control is no longer read-only, has the ability to move between Open/Closed states with the addition of the blade controls. Because this control is still essentially mapping the value of one pv, I've kept it as a single enumerated control.